features/product/viewing_products/seeing_products_lowest_price_before_the_discount.feature

```diff
@@ -1,44 +1,44 @@
 @viewing_products
-Feature: Seeing the product's lowest price from 30 days before the discount
+Feature: Seeing the product's lowest price before the discount
     In order to know whether the current discount is attractive
     As a Guest
-    I want to see a product's lowest price within 30 days before the discount
+    I want to see a product's lowest price before the discount
 
     Background:
         Given the store operates on a single channel in "United States"
 
-    @todo
+    @api
     Scenario: Not seeing the lowest price information on a product with no discount
         Given the store has a product "Wyborowa Vodka" priced at "$21.00"
         When I check this product's details
         Then I should not see information about its lowest price
 
-    @todo
+    @api
     Scenario: Seeing the lowest price information on a product that has a single discount
         Given the store has a product "Wyborowa Vodka" priced at "$37.00"
         And this product's price was discounted to "$21.00"
         When I check this product's details
-        Then I should see "$37.00" as this product's lowest price from 30 days before the discount
+        Then I should see "$37.00" as this product's lowest price before the discount
 
-    @todo
+    @api
     Scenario: Seeing the lowest price information on a product that has a discount, and then seeing the discount change
         Given the store has a product "Wyborowa Vodka" priced at "$37.00"
         And this product's price was discounted to "$21.00"
         And this product's discounted price changed to "$30.00"
         When I check this product's details
-        Then I should see "$21.00" as this product's lowest price from 30 days before the discount
+        Then I should see "$21.00" as this product's lowest price before the discount
 
-    @todo
+    @api
     Scenario: Seeing the lowest price information on a product that has a discount, over a month passes, and then seeing the discount change
         Given it is "2023-03-14" now
         And the store has a product "Wyborowa Vodka" priced at "$42.00"
         And this product's price was discounted to "$21.00"
         And it is "2023-04-15" now
         And this product's discounted price changed to "$33.00"
         When I check this product's details
-        Then I should see "$21.00" as this product's lowest price from 30 days before the discount
+        Then I should see "$21.00" as this product's lowest price before the discount
 
-    @todo
+    @api
     Scenario: Seeing the lowest price information on a product that has a discount and the one of the previous promotions ended over 30 days ago
```

Comment:
I left the "30 days ago" on purpose, as I haven't got an idea how to name it using a certain shop-configured value